docs/code/MlNetCookBook.md

```diff
@@ -306,6 +311,45 @@ var someRows = mlContext
 var featureColumns = transformedData.GetColumn<string[]>(transformedData.Schema["AllFeatures"])
 
 ```
+## How do I look at intermediate transformers as they are trained?
```

Comment:
The following code snippet is lifted verbatim from docs/samples/Microsoft.ML.Samples/Dynamic/WithOnFitDelegate.cs